tests/foreman/ui/test_adusergroup.py

```diff
@@ -92,14 +99,20 @@ def test_positive_add_admin_role(self):
                 ext_authsourceid='LDAP-' + self.ldap_server_name,
             )
             self.assertIsNotNone(self.usergroup.search(self.usergroup_name))
+            set_context(session, org=ANY_CONTEXT['org'])
```

Comment:
It is repeatable code through all the methods, but only that approach give us proper test cases isolation